src/course-unit/data/utils.js

```diff
@@ -3,196 +3,7 @@ import { camelCaseObject } from '@edx/frontend-platform';
 import { NOTIFICATION_MESSAGES } from '../../constants';
 import { PUBLISH_TYPES } from '../constants';
 
-export function getTimeOffsetMillis(headerDate, requestTime, responseTime) {
-  // Time offset computation should move down into the HttpClient wrapper to maintain a global time correction reference
-  // Requires 'Access-Control-Expose-Headers: Date' on the server response per https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#access-control-expose-headers
-
-  let timeOffsetMillis = 0;
-  if (headerDate !== undefined) {
-    const headerTime = Date.parse(headerDate);
-    const roundTripMillis = requestTime - responseTime;
-    const localTime = responseTime - (roundTripMillis / 2); // Roughly compensate for transit time
-    timeOffsetMillis = headerTime - localTime;
-  }
-
-  return timeOffsetMillis;
-}
-
-export function normalizeMetadata(metadata) {
-  const requestTime = Date.now();
-  const responseTime = requestTime;
-  const { data, headers } = metadata;
-  return {
-    accessExpiration: camelCaseObject(data.access_expiration),
-    canShowUpgradeSock: data.can_show_upgrade_sock,
-    contentTypeGatingEnabled: data.content_type_gating_enabled,
-    courseGoals: camelCaseObject(data.course_goals),
-    id: data.id,
-    title: data.name,
-    offer: camelCaseObject(data.offer),
-    enrollmentStart: data.enrollment_start,
-    enrollmentEnd: data.enrollment_end,
-    end: data.end,
-    start: data.start,
-    enrollmentMode: data.enrollment.mode,
-    isEnrolled: data.enrollment.is_active,
-    license: data.license,
-    userTimezone: data.user_timezone,
-    showCalculator: data.show_calculator,
-    notes: camelCaseObject(data.notes),
-    marketingUrl: data.marketing_url,
-    celebrations: camelCaseObject(data.celebrations),
-    userHasPassingGrade: data.user_has_passing_grade,
-    courseExitPageIsActive: data.course_exit_page_is_active,
-    certificateData: camelCaseObject(data.certificate_data),
-    entranceExamData: camelCaseObject(data.entrance_exam_data),
-    timeOffsetMillis: getTimeOffsetMillis(headers && headers.date, requestTime, responseTime),
```

Comment:
Just curious: what did we need this for, before?

Reply:
Previously, we transferred functionality from MFE Learning, so the data received from the LMS endpoint is [formatted](https://github.com/openedx/frontend-app-learning/blob/b547ada1a5e9e8c7f6eea2d510c499e64ee41785/src/courseware/data/api.js#L90) there. We decided not to refactor the code that was awaiting replacement and after switching to Studio endpoints, delete it.